account-decoder/Cargo.toml

```diff
@@ -10,11 +10,11 @@ license = "Apache-2.0"
 edition = "2021"
 
 [dependencies]
+Inflector = "0.11.4"
```

Comment:
Hahaha :eyetwitch:

Reply:
I too prefer that it was pictographicly sorted next to lowercase ell before